### PR TITLE
Node/add controlflow

### PR DESCRIPTION
**Pull request by ka-sh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,11 @@ The the markdown used to render this page is maintained [here](https://github.co
 - [Vinyl](https://github.com/gulpjs/vinyl) - Virtual file format.
 - [nvm](https://github.com/creationix/nvm) - Node Version Manager
 
+##### Control flow
+
+- [Async](https://github.com/caolan/async) - Async is a utility module which provide tools to work with synchronous javascript.
+- [q](https://github.com/kriskowal/q) - Promises library.
```
Comment (bgever) [inline README.md:160]: I believe Q is already in the list. Could you remove the other one?

**Comment by ka-sh** [comment]
You mean the one on under language? or should I remove my own q?

**Comment by bgever** [comment]
It's okay to remove the one under Language, your proposed Control Flow section makes more sense. :+1:

**Comment by ka-sh** [comment]
Okay

**Comment by ka-sh** [comment]
Done, I assume the changes will be reflected in this pull request since I can see them in the commits, or do I need to request another pull request ?

**Comment by bgever** [comment]
Thanks @ka-sh, I see the changes too, so I'll accept the PR, no need to send another one.

**Pull request merged**